packages/server/test/api-messages.js

```diff
@@ -0,0 +1,36 @@
+const { test } = require('ava')
+const { makeChannel, makeUser, testWithServer } = require('./_serverUtil')
+const fetch = require('./_fetch')
+
+let portForApiMessageTests = 26000
+
+test('POST /api/messages', t => {
+  return testWithServer(portForApiMessageTests++, async ({ server, port }) => {
```

Comment:
Don't know when you implemented this, but :+1:

Reply:
I realized that our server-based tests weren't having their servers be killed by `server.kill()` because of the test failing before it reached that point. `testWithServer` gets around that with a `try`/`finally` block.